src/rules/indentRule.ts

```diff
@@ -121,6 +128,161 @@ function walk(ctx: Lint.WalkContext<Options>): void {
     }
 }
 
+function checkIndentSize(params: {
+    ctx: Lint.WalkContext<Options>;
+    tabs: boolean;
+    size: number;
+    pos: number;
+    indentEnd: number;
+    line: string;
+    sourceFile: ts.SourceFile;
+}): boolean {
+    const ctx: Lint.WalkContext<Options> = params.ctx;
+    const tabs: boolean = params.tabs;
+    const size: number = params.size;
+    const pos: number = params.pos;
+    const indentEnd: number = params.indentEnd;
+    const line: string = params.line;
+    const sourceFile: ts.SourceFile = params.sourceFile;
+
+    const currentNodePos: number = pos + indentEnd;
+    let currentNode: ts.Node | undefined = recursiveGetNodeAt(sourceFile, currentNodePos);
+    let depth = 0;
+    if (currentNode !== undefined && inTsx(sourceFile)) {
```

Comment:
Is there a reason to do nothing at all in .tsx files? And should the same apply to .jsx?
If there is a good reason, you could consider moving this check to the top of `walk` to avoid unnecessary work.

Reply:
Added the check for jsx syntax.